aws/datadog_policy_macro.yaml

```diff
@@ -65,6 +65,48 @@ Resources:
                     ]
                 }
             }
+        def generate_cloud_cost_policy(bucket_name, report_prefix, report_name, has_cost_explorer_permissions):
+            if not bucket_name or not report_name:
+                return {"Ref" : "AWS::NoValue"}
+
+            if report_prefix:
+                data_path = f"arn:aws:s3:::{bucket_name}/{report_prefix}/{report_name}/*"
+            else:
+                data_path = f"arn:aws:s3:::{bucket_name}/{report_name}/*"
+
+            statement = [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:ListBucket"
+                    ],
+                    "Resource": f"arn:aws:s3:::{bucket_name}"
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "s3:GetObject"
+                    ],
+                    "Resource": data_path
+                }
+            ]
+
+            if has_cost_explorer_permissions:
```

Comment:
I believe `has_cost_explorer_permissions` is a string, with allowed values `true`, `false` and not a bool

Reply:
Also, should we only look to install the `DatadogCloudCostPolicy` if `has_cost_explorer_permissions` is True? I see that we do return a `AWS::NoValue` if bucket_name/report_name is empty.